### ISSUE TITLE
Document how to manually change the quota

### ISSUE THREAD

**Issue by narFnarF** [YunoHost-Apps/borgserver_ynh#64]
It's currently unclear how we're supposed to change the quota.

From what I've seen here: https://github.com/YunoHost-Apps/borgserver_ynh/issues/60 the process is:
1. Open `/home/BORG_USERNAME/.ssh/authorized_key`
2. Edit the `command="borg serve --storage-quota 200G`
3. Open `/etc/yunohost/apps/BORG_APP_NAME/settings.yml`
4. Edit the line `quota: '200G'`

Is there anything else that needs to be edited?

**Comment by narFnarF** [comment]
Might be useful too: Additional info from the officiel borg manual: https://borgbackup.readthedocs.io/en/stable/usage/serve.html#borg-serve

**Comment by narFnarF** [comment]
I could probably write some documentation/guides. Where should I do that? The readme? https://github.com/YunoHost-Apps/borgserver_ynh/blob/master/README.md
Or maybe this description file? https://github.com/YunoHost-Apps/borgserver_ynh/blob/master/doc/DESCRIPTION.md

**Comment by Gredin67** [comment]
I think you can go ahead and write these tips in https://github.com/YunoHost-Apps/borgserver_ynh/blob/master/doc/DESCRIPTION.md

I think README is auto-generated, so it should not be edited.

At some point it would be better to have these options in a config panel though.

**Comment by narFnarF** [comment]
I agree it should be configurable through the admin interface. But in the meantime, I wrote the instruction in the documentation and created a pull request: https://github.com/YunoHost-Apps/borgserver_ynh/pull/77

A maintainer of this app will need to review my changes and merge it so that my improvements can benefit everyone.
